app/Main.hs

```diff
@@ -93,12 +93,19 @@ printTermResults :: (String, TermResults) -> IO ()
 printTermResults (_, results) = do
     printMatches results $ matches results
 
+likelihoodColor :: TermResults -> Color
+likelihoodColor r =
```

Comment:
You can pattern match in the function binding without a `case` statement, like `likelihoodColor High = Red`.